### ISSUE TITLE
Change css to be semantic

### ISSUE THREAD

**Issue by AdrianoCahete** [kandebonfim/show-your-terms#6]
What `grey`class means? And `yellow`?
How about `info`? Or `warning`?

Change:

```
.red  
color: $term-color-red

.yellow  
color: $term-color-yellow

.green  
color: $term-color-green

.blue  
color: $term-color-blue

.grey
color: $term-color-grey
```

to 

```
.error
color: $term-color-red

.warning
color: $term-color-yellow

.success
color: $term-color-green

.info-alt
color: $term-color-blue

.info
color: $term-color-grey
```

This will be easy to create themes without change html.

**Comment by kandebonfim** [comment]
hmmm, nice idea, @AdrianoCahete!

Do you think this will do the job? 👇 

``` sass
.red,.error
  color: $term-color-red

.yellow,.warning
  color: $term-color-yellow

.green,.success
  color: $term-color-green

.blue,.info-alt
  color: $term-color-blue

.grey,.info
  color: $term-color-grey
```

Maybe just the `.{function}` doesn't make clear what color will be printed and the user will be required to look at the class reference every time.

**Comment by AdrianoCahete** [comment]
The problem is: If i need/want to change `.success` to print blue, for example. The `.green` class will be useless (and wrong). This might happen to themes or accessibility themes.

I think to be clear you'll need read the docs ou use the standard software's behavior.

**Comment by kandebonfim** [comment]
hmmmm, it makes sense. Do you want to open a Pull Request to fix this? I'd like to implement your idea to the project. I can take care of the changes in the project's page later.

**Comment by AdrianoCahete** [comment]
Yes, can assign to me. I'll try to do this this week, at least in the next weekend (July 2/3).
